### DISCUSSION TITLE
Pygame not working

### DISCUSSION THREAD

**Discussion by parin-th** [community/community#45931] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

When i try to pip install pygame the terminal gives an error saying the problem is in the package not pip but when i try to download other things with pip it gives me the same message can someone **_PLEASE HELP!!_*

**Comment by parin-th** [comment]
> ### Select Topic Area
> Question
> 
> ### Body
> When i try to pip install pygame the terminal gives an error saying the problem is in the package not pip but when i try to download other things with pip it gives me the same message can someone *__PLEASE HELP!!__

when i run a file that has pygame:
![image](https://user-images.githubusercontent.com/105762164/215835634-b3162a63-8366-40e7-a587-9f3d5c7038e3.png)

When I pip install pygame:
![image](https://user-images.githubusercontent.com/105762164/215836264-e47a1ffb-50a1-4469-a725-93fff61a7e40.png)

**Comment by airtower-luna** [comment]
It looks like you're looking for help with a software hosted on GitHub, I assume https://github.com/pygame/pygame. In that case you should look for support information *in that repository*, usually in the README or a file called SUPPORT. If those don't say anything, opening an issue there is usually good. :slightly_smiling_face:

That said, `sdl2-config: not found` looks like you're missing SDL libraries and tools.

**Comment by ahmadali-jamali** [comment]
You should check your pygame version. When you installed a pip line of some libraries and it does not work, it depends on your library's version.
Install this pygame on your cmd :
pip install pygame
pip install pygame==2.0.1

pip install pygame==1.9.3
pip install pygame==1.9.2b6

These are all versions of pygame.
I hope it helps you

**Comment by advayc** [comment]
if you keep having this issue after reinstalling pygame then follow these steps 

1. On VS code click view and then command pallet
2. type in select interpreter and open it
3. change the environment and select it as your interpreter
